odoo/tools/translate.py

```diff
@@ -785,7 +785,11 @@ def write_rows(self, rows):
 
 def trans_export(lang, modules, buffer, format, cr):
 
-    translations = trans_generate(lang, modules, cr)
+    reader = TranslationExporter(cr, modules=modules)
+    reader.export_translatable_records()
+    reader.export_translatable_resources()
+    translations = reader.read_rows(lang)
+
     modules = set(t[0] for t in translations)
```

Comment:
By the way, what is the point of this statement?
It should give the same module names as the ones passed in argument, shouldn't it?